integration-test/src/test/java/org/jboss/pnc/integration_new/endpoint/RestTest.java

```diff
@@ -0,0 +1,283 @@
+                .then()
+                .statusCode(200)
+                .body(JsonMatcher.containsJsonAttribute("id"));
+    }
+
+    @Test
+    @InSequence(2)
+    public void shouldGetAllProductsVersionsForSpecificProduct() {
+        given().headers(testHeaders)
+                .contentType(ContentType.JSON)
+                .port(getHttpPort())
+                .when()
+                .get(PRODUCT_REST_ENDPOINT + productId + "/versions")
+                .then()
+                .statusCode(200)
+                .body(JsonMatcher.containsJsonAttribute(FIRST_CONTENT_ID, value -> productVersionId = value));
+    }
+
+    @Test
+    @InSequence(3)
+    public void shouldGetSpecificProductsVersions() {
+        given().headers(testHeaders)
+                .contentType(JSON_PATCH)
+                .port(getHttpPort())
+                .when()
+                .get(PRODUCT_VERSION_REST_ENDPOINT + productVersionId)
+                .then()
+                .statusCode(200)
+                .body(JsonMatcher.containsJsonAttribute("id", value -> productVersionId = value));
+    }
+
+    @Test
+    @InSequence(4)
+    public void shouldGetAllProjects() {
+        given().headers(testHeaders)
+                .contentType(ContentType.JSON)
+                .port(getHttpPort())
+                .when()
+                .get(PROJECT_REST_ENDPOINT)
+                .then()
+                .statusCode(200)
+                .body(JsonMatcher.containsJsonAttribute(FIRST_CONTENT_ID, value -> projectId = value));
+    }
+
+    @Test
+    @InSequence(5)
+    public void shouldGetSpecificProject() {
+        given().headers(testHeaders)
+                .contentType(JSON_PATCH)
+                .port(getHttpPort())
+                .when()
+                .get(PROJECT_REST_ENDPOINT + projectId)
+                .then()
+                .statusCode(200)
+                .body(JsonMatcher.containsJsonAttribute("id"));
+    }
+
+    @Test
+    @InSequence(6)
+    public void shouldGetCurrentUser() {
+        given().headers(testHeaders)
+                .contentType(ContentType.JSON)
+                .port(getHttpPort())
+                .when()
+                .get(USER_REST_ENDPOINT + "current")
+                .then()
+                .statusCode(200)
+                .body(JsonMatcher.containsJsonAttribute("id"));
+    }
+
+    @Test
+    @InSequence(7)
+    public void shouldCreateNewProduct() throws IOException {
+        String rawJson = IoUtils.readFileOrResource("product", "product.json", getClass().getClassLoader());
+
+        Response response = given().headers(testHeaders)
+                .body(rawJson)
+                .contentType(ContentType.JSON)
+                .port(getHttpPort())
+                .when()
+                .post(PRODUCT_REST_ENDPOINT);
+        Assertions.assertThat(response.statusCode()).isEqualTo(201);
+
+        newProductId = response.getBody().jsonPath().get("id");
```

Comment:
can this be done in simmilar fashion to the other tests?
`.then().statusCode(201).body(JsonMatcher...`

Reply:
And the same for the following tests too.